ompi/mca/topo/treematch/topo_treematch_dist_graph_create.c

```diff
@@ -382,7 +382,6 @@ int mca_topo_treematch_dist_graph_create(mca_topo_base_module_t* topo_module,
 
     /* Centralized Reordering */
     if (0 == mca_topo_treematch_component.reorder_mode) {
-        int *k = NULL;
```

Comment:
I'm a little unsure of this change. I think this is equivalent, but I'm not sure what k is supposed to be, and this function is huge.